src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -26,6 +26,7 @@
     private String clientName = "";
```

Comment:
I'll make it consistent for now, but can I suggest that we take time to improving the naming in the implementations once we've got all of them up to the latest pac4j?  And yes should be clientNames (see my comment above about using plurals). Once vertx is out of the way, I can help out on other frameworks with simple things like renaming :)